KeytipData type error with TypeScript 3.1

<!-- Use this template for bug/error reporting only -->

### Please provide a reproduction of the bug in a codepen:
 
<!-- See https://codepen.io/FabricReact/ for a starting template -->
<!-- See http://codepen.io/dzearing/pens/public/?grid_type=list for a variety of examples -->
I can't select the TypeScript version in a codepen but I created a repo at [https://github.com/ADobin/ts31-office-fabric-react-bug](https://github.com/ADobin/ts31-office-fabric-react-bug). Run `npm install` and `npm run build` to reproduce in that repo.

### Bug Report 
 
 <!-- 
  ** If using the website, note that the website is often behind. You can find the current version the website uses at the home page (i.e. at the landing page of uifabric.io).
 -->
- __Package version(s)__: 6.81.0
- __Browser and OS versions__: (fill this out if relevant)
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No) No
 
Requested priority: (Blocking, High, Normal, Low) High
 
Products/sites affected: (if applicable)
 
#### Describe the issue:
 
Getting an error when building with TypeScript 3.1 and using any Office Fabric components:
 
#### Actual behavior:

```
node_modules/office-ui-fabric-react/lib/components/KeytipData/KeytipData.d.ts:11:55 - error TS2559: Type 'IKeytipDataProps & IRenderComponent<{}>' has no properties in common with type 'IBaseProps<any>'.

11 export declare class KeytipData extends BaseComponent<IKeytipDataProps & IRenderComponent<{}>, {}> {
                                                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

```
 
#### Expected behavior:
 
Able to compile successfully.

--- a/common/changes/office-ui-fabric-react/jg-keytip-fix-ts_2018-10-09-18-07.json
+++ b/common/changes/office-ui-fabric-react/jg-keytip-fix-ts_2018-10-09-18-07.json
@@ -1,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Keytip: Fix TS 3.1 error by extending IBaseProps.",
+      "type": "minor"
+    }
+  ],
+  "packageName": "office-ui-fabric-react",
+  "email": "jagore@microsoft.com"
+}

--- a/packages/office-ui-fabric-react/src/components/KeytipData/KeytipData.types.ts
+++ b/packages/office-ui-fabric-react/src/components/KeytipData/KeytipData.types.ts
@@ -1,6 +1,9 @@
+import { IBaseProps } from '../../Utilities';
 import { IKeytipProps } from '../../Keytip';
 
-export interface IKeytipDataProps {
+export interface IKeytip {}
+
+export interface IKeytipDataProps extends IBaseProps<IKeytip> {
   /**
    * IKeytipProps to create from this KeytipData
    * If no keytipProps are defined, a keytip won't be registered

